components/bl-pdfViewer-component/src/subcomponents.js

```diff
@@ -39,8 +65,10 @@ export function Controls(props) {
         <input
           ref={ inputRef }
           className="page-input"
-          value={ pageIndex }
-          onChange={ handlerPageChange }
+          value={ inputValue }
+          onChange={ ({ target }) => setInputValue(target.value) }
```

Comment:
I believe we could avoid using addational state for inputValue here and make this input uncotroller. Could you try to do it?